### DISCUSSION TITLE
JobRunrPro database tables customization

### DISCUSSION THREAD

**Discussion by SnehaMathankar** [jobrunr/jobrunr#552] [Q&A]
Is there any provision in JobRunrPro to customize some columns in database tables? We are in need of 2-3 additional columns in jobs table.

**Comment by patkujawa-wf** [comment]
What extra columns would you add, for what purpose? Does the jobAsJson not allow whatever flexibility you need? How would those columns be populated?

We have separate tables for things that the jobs need but are longer lived than jobs (and also indexed), which works for our use cases.

**Reply by SnehaMathankar** [reply]
Sir basically we have requirement to bind jobs to particular folder. We have to fetch folder specific schedulers. So, we require to add 2-3 columns in jobs table to store folderId & folderName. Just wanted to confirm, if we can have additional columns in jobs table through apis or any configurations in Pro version.

**Reply by patkujawa-wf** [reply]
I'm not aware of any configuration or apis that would allow that today. @rdehuyss is the maintainer, I'm just a user who's curious 🙂
